### ISSUE TITLE
[BUG] Sets Fusion alerts to informational

### ISSUE THREAD

**Issue by garybushey** [briandelmsft/SentinelAutomationModules#231]
**Describe the bug**
When a Fusion incident is created, this will change the severity to informational

**Module Name**
Name of the STAT Module containing the bug (if known)

**To Reproduce**
Hard to reproduce as Fusion has its own rules

**Expected behavior**
I would think leaving Fusion rule's severity alone would be best

**Comment by briandelmsft** [comment]
Hi @garybushey, just to get a bit more information about this I'd like to confirm a few things.  It's the playbook Triage-RelatedContent that you have running against the incident created by fusion rule is that correct which is lowering the severity?  

The core of this issue is likely the the related alerts module excludes the alerts that are attached to the incident we are triaging, as not to fire a finding on itself.  But a fusion incident is already binding multiple alerts together so we are likely not finding any other alerts as the related alerts have already been bound together.

Now, we are thinking in most cases there will be a need to build different playbooks built on top of STAT to handle different scenarios and that the Triage-Related content wouldn't be run against all incidents of all types, but rather it's built more as a sample to show you how STAT can be used to collect data and make decisions in automation.  So they way we envisioned this would be that for something like a fusion alert you may not want to lower severity or take any action at all since it's already been analyzed... perhaps you would only use STAT to provide the enrichments in this case.  To do this you'd have a separate playbook that does the enrichment only and have your automation rule point to that playbook for any incidents where you don't want an action taken on.

That being said I want to try to reproduce this and see what we can do in the related alerts module to handle this a bit differently in the case of a fusion incident, since essentially related alerts have always been found in these cases.

**Comment by garybushey** [comment]
You are correct about the module.  I have modified my automation rule to ignore the Fusion rules so that should take care of my issue.

**Comment by piaudonn** [comment]
I suggest the issue is renamed to reflect that the challenge is getting the entities on Fusion incidents.

**Comment by briandelmsft** [comment]
I reproduced a fusion incident via an atypical travel + credential harvesting tool and the Entities array was empty.  I'm not sure if this is the case for **_all_** fusion incidents but this would be a problem for any automation that uses entities data (almost all templates in the Sentinel github use entities data).

STAT is effectively working as intended as it takes the input entities (none) and checks them for related alerts, TI, etc but since there's no inputs it can't find anything.  If no fusion incidents actually surface their entities to automation I see this as a problem with the fusion incidents, not STAT.

![image](https://user-images.githubusercontent.com/68655382/153940549-a182f1ea-66bf-4061-8004-0a4bf4e1ef14.png)

Even the built in connector to get accounts didn't handle this and find the entities via the linked alerts and returned an empty array

![image](https://user-images.githubusercontent.com/68655382/153941010-40191bdf-6e3f-4402-b6a6-46ed76c3dc80.png)

The GUI shows the entities but the incident trigger does not

![image](https://user-images.githubusercontent.com/68655382/153941334-9292d018-de3a-47e7-bb30-5a57ade78d95.png)


@garybushey can you confirm if the fusion incident you processed with STAT similarly has no entities data in the incident creation rule step of the previous logic app run

**Comment by garybushey** [comment]
The Fusion incident I have used the "Office policy tampering" and the "Rare and potentially high-risk Office operations" rules and they DO have entities (username and IP Address)

**Comment by briandelmsft** [comment]
Ok so I think we have this one covered now,

* PR #232 updating the docs and deployment to better highlight the sample playbook as such
* PR #233 adding handling to the Base Module so if not entities are presented we intentionally fail since we can't triage without entities
* PR #234 will consider alerts linked to a fusion incident as 'related' when normally this module excludes alerts linked to the incident being analyzed
